coding-concepts.js

```diff
@@ -10,7 +10,9 @@ const colors = ["tangerine", "magenta", "lilac", "daffodil"]
 // console.log(colors.push("indigo"))
 
 // a) Your answer:
-// b) Verify and explain:
+// I believe that .push will make so that the variable "indigo" will be added to the array at the end. Thus, the output will be tangerine, magenta, lilac, daffodil, indigo.
+
+// b) Verify and explain: It would seem that instead of adding the term at the end, the code still added a value which makes it total of 5 values but only output the number 5.
```

Comment:
Index and length are tricky. Index is the location of a value. Length tells you how many values are in a datatype. `.push(value)` will give add an additional value to this array that has 4 currently. Therefore, adding `indigo` makes 5 values in the array now. Therefore the `.push(value)` method is output the amount of values in the element.